src/AddressRegistry.Api.BackOffice/AcceptedWithETagResult.cs

```diff
@@ -13,6 +13,12 @@ public class AcceptedWithETagResult : AcceptedResult
         private readonly List<string> _tags;
         public string ETag { get; }
 
+        public AcceptedWithETagResult(string eTag, params string[] tags)
+        {
+            _tags = new List<string>(tags);
+            ETag = eTag;
+        }
+
```

Comment:
Doesn't really makes sense to just return the etag.
I would return just `Accepted` with location, no etag. When they follow the location they will see a `410 Gone`.

Reply:
Changed.